.gitignore

```diff
@@ -11,12 +11,14 @@ npm-debug.log
 node_modules
 v1/node_modules/*
 v1/npm-debug.log
-
+.github
 v1/utilities/scripts/credentials.py
+.idea
+venv
 
 credentials.ts
 !v1/credentials.ts
-!v2/credentials.ts
+v2/credentials.ts
```

Comment:
The reason the Gitignore was structured this way for credentials is that the `v2/credentials.ts` was essentially a re-export of the "main" credentials file. Now that there's only v2, it's less important. But we should definitely ensure that two credentials files are not needed. And ideally, all credentials would be specified as environment variables or in an `.env` file

Reply:
Right now the code references v2/credentials.ts. I commented out the rest of the other cred URI's. Will create .env once I can get a query to successfully execute from macrostrat_two in PG